paddle/majel/allocation.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <memory>
+
+#include "place.h"
```

Comment:
Let us use full include path name -- `#include "paddle/majel/place.h"`.  I know everyone might have a different idea about this, but let us just unify. Thanks.